src/util.ts

```diff
@@ -18,6 +18,10 @@ export function extractSiteName(site: WebSiteModels.Site): string {
     return isSiteDeploymentSlot(site) ? site.name.substring(0, site.name.lastIndexOf('/')) : site.name;
 }
 
+export function extractSiteScmSubDomainName(site: WebSiteModels.Site): string {
```

Comment:
You don't have to change it for this PR, but just FYI we're trying to move away from using `site` directly and instead using the `SiteWrapper` class from our appservice npm package. In this case, I believe `SiteWrapper.appName` duplicates this logic https://github.com/Microsoft/vscode-azuretools/blob/master/appservice/src/SiteWrapper.ts#L56

(We created that npm package because the Azure Functions extension uses a lot of the same code and we didn't want it to be duplicated)

Reply:
Sounds good. Will add an issue to change to using SiteWrapper later.

Reply:
Added #240